Library/Formula/sdl_ttf.rb

```diff
@@ -18,8 +18,13 @@ class SdlTtf < Formula
   depends_on "sdl"
   depends_on "freetype"
 
+  # Fix broken TTF_RenderGlyph_Shaded()
+  # https://bugzilla.libsdl.org/show_bug.cgi?id=1433
+  patch :DATA
+
   def install
     ENV.universal_binary if build.universal?
+    inreplace "SDL_ttf.pc.in", "@prefix@", HOMEBREW_PREFIX
```

Comment:
I'm also not comfortable with `HOMEBREW_PREFIX` either, but there seems to be a historical reason for this: d37ab41. I just wanted to make it consistent with other SDL packages. Do you think we better leave it as a separate PR then?

And it wouldn't be easy to patch only this specific location with `inreplace`. The exact same `for` loop statement (with legit use of both `pixmap` and `bitmap`) appears in another part of the code multiple times.